### DISCUSSION TITLE
REST API Project Documentation Incomplete

### DISCUSSION THREAD

**Discussion by LGRmac** [exogee-technology/graphweaver#956] [Q&A] [answered]
Hi all,

Looks like the documentation for connecting to a REST API is incomplete. Following the documentation to completion does not lead to a successful connection between GraphWeaver and a REST API. (https://graphweaver.com/docs/connect-to-a-rest-api)
- The REST API is not called from the fetch function from the 'rest-client.ts' file - as far as I can tell.
- A filename and location for the final code snippet of the page, is not given. Hence this code is orphaned and is never run.
- According to VS Code, the 'got' module cannot be found from the 'rest-client.ts' page. This could be my environment, but again all instructions were followed in the documentation and it does not mention to install this module manually.

The server successfully runs but there is no indication that the API is called - and no data is returned like in the Demo on the GraphWeaver website. (https://demo.graphweaver.com/)

Is someone able to update the documentation to provide necessary steps to get the API connection to work correctly.

**Comment by thekevinbrown** [comment]
Hey @LGRmac, thank you for flagging this. I'll run through these instructions myself and make sure they're up to date. We can also share the demo code running at https://demo.graphweaver.com if that'd be helpful.

**Reply by LGRmac** [reply]
Thanks @thekevinbrown - I opened an issue and added more specific details. Happy to provide more explanation, if needed.
Yes would be helpful to see the code running the demo.

The specific project I am trying to create is a proof of concept that will be brining in data from two REST APIs - so it would be nice to see an example of how this is meant to work.

**Reply by LGRmac** [reply, marked as answer]
https://github.com/exogee-technology/graphweaver/issues/967
